e2e/test/Helpers/templates/FaultInjection.cs

```diff
@@ -39,17 +39,17 @@ public static class FaultInjection
         public const string FaultCloseReason_Boom = "boom";
         public const string FaultCloseReason_Bye = "byebye";
 
-        public const int DefaultDelayInSec = 5; // Time in seconds after service initiates the fault.
-        public const int DefaultDurationInSec = 5; // Duration in seconds
-        public const int LatencyTimeBufferInSec = 10; // Buffer time waiting fault occurs or connection recover
+        public static readonly TimeSpan DefaultDelayInSec = TimeSpan.FromSeconds(5); // Time in seconds after service initiates the fault.
```

Comment:
Although tangent to the retry policy related changes, I am also updating our E2E tests to use TimeSpan instead of int, to represent time. (I spent longer than I should have trying to identify a test failure, only to realize the test was setting time time in ms, while the service was expecting the time in secs.